packages/EasyActivity/src/Bridge/Symfony/Serializers/SymfonyActivitySubjectDataSerializer.php

```diff
@@ -48,7 +50,7 @@ public function serialize(array $data, ActivitySubjectInterface $subject): ?stri
         $context = [];
 
         foreach ($data as $key => $value) {
-            if ($allowedProperties !== null
+            if ($allowedProperties !== []
```

Comment:
This change makes me think we shouldn't make the method return value non-nullable. What if someone wants to explicitly set the list of allowed properties to an empty one (meaning "log nothing")? `allowedProperties === null` seems more explicit to me, as it is read as "allowed properties are not defined".

Reply:
done